packages/in_app_purchase/in_app_purchase_ios/ios/Classes/FIAObjectTranslator.m

```diff
@@ -193,4 +209,24 @@ + (NSDictionary *)getMapFromSKStorefront:(SKStorefront *)storefront
   return map;
 }
 
++ (SKPaymentDiscount *)getSKPaymentDiscountFromMap:(NSDictionary *)map {
```

Comment:
Is this method used?

Reply:
No it was not used, however it should have been used. I have added usage (including tests) in commit 6abe36d.